Add post about joining the Open Source Collective

rth opened the pull request:
Closes https://github.com/pyodide/pyodide-blog/issues/2

The tone of voice is maybe a but too formal for a blog. [Similar announcement at PyPy](https://morepypy.blogspot.com/2020/08/pypy-is-on-open-collective.html) for instance.

Post preview: https://deploy-preview-6--pyodide-blog.netlify.app//posts/pyodide-joins-open-source-collective/  (the netlify preview uses the wrong subdomain when you click on links)

**Planned publication date:** 2021/11/16

rth commented on content/posts/pyodide-joins-open-source-collective.md:
```diff
@@ -0,0 +1,47 @@
+
+We are now accepting donations at:
+ - the OSC project page: [opencollective.com/pyodide](https://opencollective.com/pyodide)
+ - or via Github Sponsors: [github.com/sponsors/pyodide](https://github.com/sponsors/pyodide)
```
As mentioned in https://github.com/pyodide/pyodide/issues/1916#issuecomment-966209388 I made the request for validating the account yesterday, and it takes a few days to validate it. If we don't hear from them soon, we can also remove this.

rth commented on content/posts/pyodide-joins-open-source-collective.md:
```diff
@@ -0,0 +1,47 @@
+date: 2021-11-10T13:13:04+01:00
+draft: true
+tags: ["announcement"]
+author: "Roman Yurchak on behalf of the Pyodide team"
```
We need to decide how we formulate this in project announcements blog posts. We could just put "The Pyodide team" as the author, but I feel the person who writes the post should still appear somewhere. Also in practice most of these would likely be co-written / changed through review, so here it would likely be Hood and me?

rth commented on config.yaml:
```diff
@@ -27,7 +27,7 @@ params:
     disableThemeToggle: false
 
     ShowReadingTime: true
-    ShowShareButtons: true
```
Removed because I find the button to share on 10 social media a bit too much, but we should include more prominently social media page somewhere (right now they are only in the home page introduction).

hoodmane replied:
Yeah I was looking at a test output and all the share buttons bugged me too.

hoodmane commented on content/posts/pyodide-joins-open-source-collective.md:
```diff
@@ -0,0 +1,47 @@
+Collective (OSC)](https://www.oscollective.org/), which will act as project's
+fiscal host.
+
+The Pyodide project, that compiles Python and the scientific stack to
+WebAssembly, was created in 2018 by Michael
```
Flows better as two separate sentences IMO.
```suggestion
The Pyodide project compiles Python and the scientific stack to WebAssembly using Emscripten. 
Pyodide was created in 2018 by Michael
```

hoodmane commented on content/posts/pyodide-joins-open-source-collective.md:
```diff
@@ -0,0 +1,47 @@
+The Pyodide project, that compiles Python and the scientific stack to
+WebAssembly, was created in 2018 by Michael
+Droettboom at Mozilla as part of the [Iodide
+project](https://github.com/iodide-project/iodide). Later in 2021 [Pyodide
+become an indepenent open-source
+project](https://hacks.mozilla.org/2021/04/pyodide-spin-out-and-0-17-release/),
+and the current annoucement completes this transition. By selecting OSC as our
+fiscal host, Pyodide will be able to recieve donnations and accepts
+sponsorships which would help long term project sustainability.
```
Fix typos and improve flow of last sentence
```suggestion
project](https://github.com/iodide-project/iodide). Later in 2021 [Pyodide
become an independent open-source
project](https://hacks.mozilla.org/2021/04/pyodide-spin-out-and-0-17-release/),
and the current announcement completes this transition. Pyodide will now be able to receive donations and accept
sponsorships though OSC, which will help long term project sustainability.
```

hoodmane commented:
> The tone of voice is maybe a but too formal for a blog. Similar announcement at PyPy for instance.

Yeah the PyPy announcement is definitely less formal. The main reason is that they editorialize a bit on stuff -- they have some opinions in there. The easiest way to decrease formality would be to add a few of our own.

rth commented:
@hoodmane Could you have a look? Tried to make it a bit less formal. Also,
 - added you as a coauthor
 - removed GH sponsors link, since we have no news so far
 - added a paragraph about starting this blog.

I should really try to reduce the number of typos in my writing; it's embarrassing. Maybe I should investigate if there are some grammar checkers for vim, the default one is pretty basic.

rth commented:
I would like to publish this tomorrow, so please let me know if you have any other comments.
